### PR TITLE
Translate `ui` extension

### PR DESCRIPTION
**Pull request by Fan2Shrek**
I'm not sure about the translations for the `<title>* Control</title>` elements.

### CODE REVIEW

File: reference/ui/ui.area.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ <title>Area</title>
+ <titleabbrev>UI\Area</titleabbrev>
+
+ <partintro>
+
+<!-- {{{ UI\Area intro -->
+  <section xml:id="ui-area.intro">
+   &reftitle.intro;
+   <para>
+    Une Area représente un canevas sur lequel on peut dessiner et répondre aux événements de souris et de clavier.
```
Comment (devnexen) [inline reference/ui/ui.area.xml:14]: nit: it s a bit inconsistent, in other files you do `french word` (`english word`) e.g. ui.controls.box.xml

File: reference/ui/ui.controls.editablecombo.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ <title>boîte de combinaison éditable de Controle</title>
+ <titleabbrev>UI\Controls\EditableCombo</titleabbrev>
+
+ <partintro>
+
+<!-- {{{ UI\Controls\EditableCombo intro -->
+  <section xml:id="ui-controls-editablecombo.intro">
+   &reftitle.intro;
+   <para>
+    Un boîte de combinaison éditable (Editable Combo) est un Combo qui permet à l'utilisateur de saisir des options personnalisées
```
Comment (devnexen) [inline reference/ui/ui.controls.editablecombo.xml:14]: `Une boîte`

File: reference/ui/ui.controls.multilineentry.xml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-multilineentry" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Une entré multiligne de Controle</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.multilineentry.xml:5]: `une entrée`

File: reference/ui/ui.controls.radio.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-radio" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Radio de Controle</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.radio.xml:5]: nit: `Contrôle`

File: reference/ui/ui.controls.separator.xml [hunk 5]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-separator" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Séparateur de Controle</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.separator.xml:5]: ditto

File: reference/ui/ui.controls.progress.xml [hunk 6]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-progress" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Contrôle de Progression de Controle</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.progress.xml:5]: Sounds a bit odd to me here wdyt ? `Contrôle de progression` is enough IMHO

File: reference/ui/ui.controls.slider.xml [hunk 7]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-slider" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Glissière de Control</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.slider.xml:5]: Like you did in other files `Glissiêre de Contrôle (Slider Control)` for example

File: reference/ui/ui.controls.spin.xml [hunk 8]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 4d17b7b4947e7819ff5036715dd706be87ae4def Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<reference xml:id="class.ui-controls-spin" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Boîte de rotation de Control</title>
```
Comment (devnexen) [inline reference/ui/ui.controls.spin.xml:5]: ditto

**Comment by devnexen** [comment]
General note: try to make it more consistent throughout.

File: reference/ui/ui.controls.slider.xml (hunk 7 above)
Comment (devnexen) [reply reference/ui/ui.controls.slider.xml]: Just title, titleabbrev seems to be the namespaces.

Comment (Fan2Shrek) [reply reference/ui/ui.controls.slider.xml]: I wasn't referring to the title abbreviation; I meant the first paragraph (around line 15). Sorry for the misunderstanding.

Comment (devnexen) [reply reference/ui/ui.controls.slider.xml]: just the intro then !

File: reference/ui/ui.point.xml [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+ <title>Représente une position (x,y)</title>
+ <titleabbrev>UI\Point</titleabbrev>
+
+ <partintro>
+
+<!-- {{{ UI\Point intro -->
+  <section xml:id="ui-point.intro">
+   &reftitle.intro;
+   <para>
+    Les point sont utilisés dans toute l'interface utilisateur pour représenter des coordonnées sur un écran, un contrôle ou une zone.
```
Comment (devnexen) [inline reference/ui/ui.point.xml:14]: Les points

File: reference/ui/ui.area.xml [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+     <term><constant>UI\Area::Super</constant></term>
+     <listitem>
+      <para></para>
+     </listitem>
+    </varlistentry>
+
+    <varlistentry xml:id="ui-area.constants.down">
+     <term><constant>UI\Area::Down</constant></term>
+     <listitem>
+      <para>Shall be set in the modifiers passed to key and mouse events</para>
```
Comment (devnexen) [inline reference/ui/ui.area.xml:117]: Forgotten translation, same below

**Review by devnexen — APPROVED** [review]

**Pull request merged**